packages/showcases/core/csf/components/actions/dropdown.csf.js

```diff
@@ -7,3 +7,41 @@ export const parameters = {
     url: 'https://www.figma.com/file/zDZIyayUlr1yTWrsi7cFoo/?node-id=2421%3A3',
   },
 };
+
+export const argTypes = {
+  label: {
```

Comment:
can you please use `labelText` instead as we did with select, text input, checkbox, radio button and toggle? Thanks.